plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShift3ResourceUtils.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v 1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.reddeer.requirement;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+
+import org.eclipse.osgi.util.NLS;
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.tools.openshift.common.core.connection.ConnectionURL;
+import org.jboss.tools.openshift.common.core.connection.ConnectionsRegistrySingleton;
+import org.jboss.tools.openshift.common.core.utils.StringUtils;
+import org.jboss.tools.openshift.core.connection.Connection;
+import org.jboss.tools.openshift.reddeer.exception.OpenShiftToolsException;
+
+import com.openshift.restclient.ResourceKind;
+import com.openshift.restclient.model.IProject;
+import com.openshift.restclient.model.IResource;
+
+/**
+ * @author adietish@redhat.com
+ */
+public class OpenShift3ResourceUtils {
+	
+	private static Logger LOGGER = new Logger(OpenShift3ResourceUtils.class);
+
+	public static Connection getConnection(String connectionUrlString) {
```

Comment:
Add javadoc, because it is not clear enough without description what form should the URL have. I have to look in other places in code, where its used, that it should look like "https://openshift-dev@10.1.2.2:8443".